pyproject.toml

```diff
@@ -15,8 +15,9 @@ dependencies = [
     "numpy>=1.24.4, <3.0.0; python_version >= '3.11'",
     "numpy>=1.24.4, <=2.2.6; python_version == '3.10'",
     "numpy>=1.24.4, <=2.0.2; python_version <= '3.9'",
-    "pgvector>=0.2.5, <1.0.0",
-    "SQLAlchemy[asyncio]>=2.0.25, <3.0.0"
+    "pgvector>=0.2.5, <0.4.0",
+    "SQLAlchemy[asyncio]>=2.0.25, <3.0.0",
```

Comment:
We may not need all these dependencies any more since they are being pulled in from langchain-postgres. Though it might be good to keep the numpy deps, since we may handle them better than lc-pg.